grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -296,6 +297,14 @@ class ShipmentItem implements Comparable, Serializable {
         return quantityToInvoiceInStandardUom > 0
     }
 
+    OrderItem getOrderItem() {
+        return orderItems[0]
```

Comment:
I'm wondering why this relationship is many-to-many

Reply:
I just made this same comment on the other pr...